desktop/cmp/loadingindicator/LoadingIndicator.js

```diff
@@ -0,0 +1,116 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2019 Extremely Heavy Industries Inc.
+ */
+
+import PT from 'prop-types';
+import {Component} from 'react';
+import {elemFactory, HoistComponent} from '@xh/hoist/core';
+import {hbox} from '@xh/hoist/cmp/layout';
+import {div} from '@xh/hoist/cmp/layout/Tags';
+import {PendingTaskModel} from '@xh/hoist/utils/async';
+import {spinner} from '@xh/hoist/kit/blueprint';
+import {withDefault} from '@xh/hoist/utils/js';
+import {truncate} from 'lodash';
+
+import './LoadingIndicator.scss';
+
+
+/**
+ * A minimal / unobtrusive LoadingIndicator displaying an optional spinner and/or message to signal
+ * that a longer-running operation is in progress, without using a modal Mask. Can be explicitly
+ * shown or bound to a PendingTaskModel.
+ *
+ * Note that the Panel component's `loadingIndicator` prop provides a common and convenient way to
+ * add an indicator to a Panel without needing to manually create or manage this component.
+ */
+@HoistComponent
+export class LoadingIndicator extends Component {
+
+    static modelClass = PendingTaskModel;
+
+    static propTypes = {
+
+        /** Position of the indicator relative to its containing component. */
+        corner: PT.oneOf(['tl', 'tr', 'bl', 'br']),
+
+        /** True to display the indicator. */
+        isDisplayed: PT.bool,
+
+        /**  Max characters allowed in message, after which it will be elided. Default 30. */
+        maxMessageLength: PT.number,
+
+        /** Optional text to be displayed - can also be sourced from bound PendingTaskModel. */
+        message: PT.string,
+
+        /** Optional model for reactively showing the indicator while tasks are pending. */
+        model: PT.instanceOf(PendingTaskModel),
+
+        /** Click handler. **/
+        onClick: PT.func,
```

Comment:
Do we have a use-case for this?  Assume we do (and it does not add any meaningful complexity).  Just wanted to be sure.

Reply:
I would be up with getting rid of this here and for mask (which is where it was copied from).
I think there are issues with its implementation (IIRC, Yana found it not working on mask) and its just cruft.

Reply:
Removed  onclick prop from mask and loadingIndicator in desktop and mobile.